### ISSUE TITLE
The repository 'https://dl.yarnpkg.com/debian stable InRelease' is not signed 

### ISSUE THREAD

**Issue by m1nyasha** [getsentry/self-hosted#1932]
### Self-Hosted Version

23.1.0

### CPU Architecture

x64

### Docker Version

20.10.17

### Docker Compose Version

1.29.2

### Steps to Reproduce

I'm used Ubuntu 20.04.4 LTS (my VDS) and Ubuntu 22.04.1 LTS (local). Everywhere the error is repeated.

1. clone release [23.1.0](https://github.com/getsentry/self-hosted/releases/tag/23.1.0)
2. start installation (install.sh)

### Expected Result

The installation should be successful.

### Actual Result

```
...
#35 [symbolicator-cleanup-self-hosted-local 4/5] RUN apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*
#35 22.69 Get:5 http://deb.debian.org/debian-security bullseye-security/main amd64 Packages [216 kB]
#35 22.81 Get:6 http://deb.debian.org/debian bullseye-updates/main amd64 Packages [14.6 kB]
#35 ...

#36 [snuba-cleanup-self-hosted-local 4/5] RUN apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*
#36 20.22 Get:8 http://deb.debian.org/debian-security bullseye-security/main amd64 Packages [216 kB]
#36 20.37 Get:9 http://deb.debian.org/debian bullseye-updates/main amd64 Packages [14.6 kB]
#36 20.93 Reading package lists...
#36 21.23 W: GPG error: https://dl.yarnpkg.com/debian stable InRelease: The following signatures were invalid: EXPKEYSIG 23E7166788B63E1E Yarn Packaging <yarn@dan.cx>
#36 21.23 E: The repository 'https://dl.yarnpkg.com/debian stable InRelease' is not signed.
#36 ERROR: executor failed running [/bin/sh -c apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*]: exit code: 100

#35 [symbolicator-cleanup-self-hosted-local 4/5] RUN apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*
#35 CANCELED

#38 [sentry-cleanup-self-hosted-local 5/5] COPY entrypoint.sh /entrypoint.sh
#38 CANCELED
------
 > [snuba-cleanup-self-hosted-local 4/5] RUN apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*:
#0 1.740 Get:3 http://deb.debian.org/debian-security bullseye-security InRelease [48.4 kB]
#0 1.907 Get:4 http://deb.debian.org/debian bullseye-updates InRelease [44.1 kB]
#0 2.134 Get:5 http://deb.debian.org/debian bullseye/main amd64 Packages [8183 kB]
#36 10.91 Get:6 https://deb.nodesource.com/node_19.x bullseye InRelease [4586 B]
#36 11.68 Get:7 https://deb.nodesource.com/node_19.x bullseye/main amd64 Packages [774 B]
#36 20.22 Get:8 http://deb.debian.org/debian-security bullseye-security/main amd64 Packages [216 kB]
#36 20.37 Get:9 http://deb.debian.org/debian bullseye-updates/main amd64 Packages [14.6 kB]
#36 20.93 Reading package lists...
#36 21.23 W: GPG error: https://dl.yarnpkg.com/debian stable InRelease: The following signatures were invalid: EXPKEYSIG 23E7166788B63E1E Yarn Packaging <yarn@dan.cx>
#36 21.23 E: The repository 'https://dl.yarnpkg.com/debian stable InRelease' is not signed.
------
failed to solve: executor failed running [/bin/sh -c apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*]: exit code: 100
Error in build-docker-images.sh:8.
'$dcb --force-rm $service' exited with status 17
-> ./install.sh:main:29
--> build-docker-images.sh:source:8

```

**Comment by KenguruKleo** [comment]
The same issue

**Comment by astappiev** [comment]
Similar issues
```
#8 0.853 Get:5 https://dl.yarnpkg.com/debian stable InRelease [17.1 kB]
#8 1.056 Get:6 http://deb.debian.org/debian bullseye/main amd64 Packages [8183 kB]
#8 1.308 Get:7 http://deb.debian.org/debian-security bullseye-security/main amd64 Packages [216 kB]
#8 1.548 Get:8 http://deb.debian.org/debian bullseye-updates/main amd64 Packages [14.6 kB]
#8 1.870 Err:5 https://dl.yarnpkg.com/debian stable InRelease
#8 1.870   The following signatures were invalid: EXPKEYSIG 23E7166788B63E1E Yarn Packaging <yarn@dan.cx>
#8 2.322 Get:9 https://deb.nodesource.com/node_19.x bullseye/main amd64 Packages [774 B]

#8 3.553 W: GPG error: https://dl.yarnpkg.com/debian stable InRelease: The following signatures were invalid: EXPKEYSIG 23E7166788B63E1E Yarn Packaging <yarn@dan.cx>
#8 3.553 E: The repository 'https://dl.yarnpkg.com/debian stable InRelease' is not signed.
------

#8 CANCELED
failed to solve: executor failed running [/bin/sh -c apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*]: exit code: 100
Error in build-docker-images.sh:8.
'$dcb --force-rm $service' exited with status 17
-> ./install.sh:main:29
--> build-docker-images.sh:source:8
```

**Comment by astappiev** [comment]
Dirty fixed by adding `RUN rm -f /etc/apt/sources.list.d/yarn.list` in `cron/Dockerfile` before `RUN apt-get update...`

**Comment by shopkueche** [comment]
> Dirty fixed by adding `RUN rm -f /etc/apt/sources.list.d/yarn.list` in `cron/Dockerfile` before `RUN apt-get update...`
> 
> As the command only needs to install `cron` package, we don't need the `yarn` repository here.

Thanks. This one did it.

**Comment by hubertdeng123** [comment]
Seeing this as well, looking into it

**Comment by hubertdeng123** [comment]
This has now been fixed [here](https://github.com/getsentry/snuba/pull/3648), seems to be working for me now.

**Comment by Sebi94nbg** [comment]
@hubertdeng123 I guess we need a new release / version to get this working with 23.1.0 or will it just work when we re-try it with the same version?

If a new release / version is required: When will this be available?

This issue is currently blocking fresh installations and updates of existing installations.

**Comment by chadwhitacre** [comment]
We'll cut 23.1.1 release tomorrow, I made a ticket https://github.com/getsentry/self-hosted/issues/1935 if you want to follow along.

**Comment by chadwhitacre** [comment]
I'm gonna go ahead and reopen this, we'll close it with the 23.1.1 release.

**Comment by Sebi94nbg** [comment]
I can confirm, that this issue has been fixed with the [release 23.1.1](https://github.com/getsentry/self-hosted/releases/tag/23.1.1).

I could successfully update my Sentry.

**Comment by chadwhitacre** [comment]
Thanks for reporting back @Sebi94nbg, calling this good. 👍
